package/kedro_viz/server.py

```diff
@@ -475,15 +476,15 @@ def format_pipeline_data(
             nodes[node_id]["pipelines"].append(pipeline_key)
 
 
-def _get_dataset_node(node_id, namespace):
+def _get_dataset_data_params(node_id, namespace):
```

Comment:
Why `node_id` argument is not used in this function?

Reply:
Mmm, seems like `node_id` argument was not used before this PR as well. I've removed it.